src/twisted/internet/defer.py

```diff
@@ -2238,13 +2238,9 @@ def _releaseAndReturn(self, r: _T) -> _T:
         self.release()
         return r
 
-    # You might wonder: "WTF is self_319AA2A8B18F4B8EA296D75F279EB07F?"
-    # It's self_ + a GUID, which is to say: "it's not a string that will ever
-    # be used as a name in kwargs".
-    # Positional-only arguments, starting in Python 3.8, would be a better
-    # alternative.
     def run(
-        self_319AA2A8B18F4B8EA296D75F279EB07F: _ConcurrencyPrimitiveT,
+        self: _ConcurrencyPrimitiveT,
+        /,
         f: Callable[..., _SelfResultT],
```

Comment:
this would be better, but would be a breaking change
```suggestion
        f: Callable[..., _SelfResultT], 
        /,
```